Tests/Pcap++Test/TestDefinition.h

```diff
@@ -64,6 +64,7 @@ PTF_TEST_CASE(TestTcpReassemblySanity);
 PTF_TEST_CASE(TestTcpReassemblyRetran);
 PTF_TEST_CASE(TestTcpReassemblyMissingData);
 PTF_TEST_CASE(TestTcpReassemblyOutOfOrder);
+PTF_TEST_CASE(TestTcpReassemblyOutOfOrderManuallyCloseConnOnMesgReady);
```

Comment:
Can we make the test name shorter?
Maybe: `TestTcpReassemblyOOOCloseConn`

Reply:
Is `TestTcpReassemblyOutOfOrderWithManualClose` fine?

Reply:
We should probably change that, but we currently have a hard-coded limit of 35 characters for test names, here is an example: https://github.com/seladb/PcapPlusPlus/blob/0c75603db71a9bef02803e9b0893a5074fcbd5da/Tests/PcppTestFramework/PcppTestFrameworkRun.h#L99

Tests that have a longer name will result in a weird UI print, something like:
```
...
...
TestTcpReassemblyMalformedPkts     : PASSED
TestTcpReassemblyMultipleConns     : PASSED
TestTcpReassemblyIPv6              : PASSED
TestTcpReassemblyIPv6MultConns     : PASSED
TestTcpReassemblyIPv6_OOO          : PASSED
TestTcpReassemblyCleanup           : PASSED
TestTcpReassemblyMaxOOOFrags       : PASSED
TestTcpReassemblyMaxSeq            : PASSED
TestTcpReassemblyDisableOOOCleanup : PASSED
TestTcpReassemblyOutOfOrderWithManualClose : PASSED
...
```